packages/sit-onyx/src/components/OnyxAvatarStack/OnyxAvatarStack.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import OnyxAvatar from "../OnyxAvatar/OnyxAvatar.vue";
+import { AVATAR_TYPES } from "../OnyxAvatar/types";
+import OnyxAvatarStack from "./OnyxAvatarStack.vue";
+
+/**
+ * An avatar stack can be used to display a list of avatars (e.g. contributors for a project etc.).
+ */
+const meta: Meta<typeof OnyxAvatarStack> = {
+  title: "components/AvatarStack",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxAvatarStack,
+    events: [],
+    argTypes: {
+      default: { control: { disable: true } },
+    },
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxAvatarStack>;
+
+/**
+ * This example shows a default avatar stack.
+ */
+export const Default = {
```

Comment:
technically, you can actually simulate the "+5" features, the initials avatar works with "+x" 😜
![image](https://github.com/SchwarzIT/onyx/assets/151019977/8559a955-a860-41a9-8e2d-c3383b891d5b)

(just not >9 ^^)

Reply:
True :D I will ask Jannick tomorrow for a "real" implementation, I can imagine that it has another background color then